src/library/Uno.Toolkit.Material/Styles/Controls/v2/Card.xaml

```diff
@@ -6,41 +6,41 @@
 
     <!-- Card Brushes Resources -->
     <!-- Elevated -->
-    <StaticResource x:Key="M3MaterialElevatedCardBackground"
+    <StaticResource x:Key="MaterialElevatedCardBackground"
 	                ResourceKey="SurfaceBrush" />
 
     <!-- Filled -->
-    <StaticResource x:Key="M3MaterialFilledCardBackground"
+    <StaticResource x:Key="MaterialFilledCardBackground"
 	                ResourceKey="SurfaceVariantBrush" />
     
     <!-- Outlined -->
-    <StaticResource x:Key="M3MaterialOutlinedCardBackground"
+    <StaticResource x:Key="MaterialOutlinedCardBackground"
 	                ResourceKey="SurfaceBrush" />
-    <StaticResource x:Key="M3MaterialOutlinedCardBorderBrush"
+    <StaticResource x:Key="MaterialOutlinedCardBorderBrush"
 	                ResourceKey="OutlineBrush" />
 
     <!--  Card Other Resources  -->
     <!-- Double -->
-    <x:Double x:Key="M3MaterialCardMinHeight">72</x:Double>
-    <x:Double x:Key="M3MaterialCardMaxWidth">344</x:Double>
+    <x:Double x:Key="MaterialCardMinHeight">72</x:Double>
+    <x:Double x:Key="MaterialCardMaxWidth">344</x:Double>
 
     <!-- Thickness -->
-    <Thickness x:Key="M3MaterialCardPadding">16</Thickness>
-    <Thickness x:Key="M3MaterialCardBorderThickness">1</Thickness>
+    <Thickness x:Key="MaterialCardPadding">16</Thickness>
+    <Thickness x:Key="MaterialCardBorderThickness">1</Thickness>
 
     <!-- Corner Radius -->
-    <CornerRadius x:Key="M3MaterialCardCornerRadius">12</CornerRadius>
+    <CornerRadius x:Key="MaterialCardCornerRadius">12</CornerRadius>
 
     <!--  Card Elevation Variables  -->
-    <x:Double x:Key="M3MaterialCardElevation">6</x:Double>
-    <Thickness x:Key="M3MaterialCardElevationMargin">6</Thickness>
+    <x:Double x:Key="MaterialCardElevation">6</x:Double>
+    <Thickness x:Key="MaterialCardElevationMargin">6</Thickness>
 
     <!--  Card Templates  -->
     <DataTemplate x:Key="M3DefaultHeaderContentTemplate">
         <TextBlock Text="{Binding}"
                    MaxLines="1"
                    Margin="16,16,16,0"
-                   Style="{ThemeResource TitleMedium}" />  
+                   Style="{ThemeResource MaterialTitleMedium}" />  
```

Comment:
```suggestion
                   Style="{ThemeResource TitleMedium}" />  
```

Reply:
i actually dont have an opinion about this one; both are fine to me

Reply:
🤦 I was running on autopilot and forgot this wouldn't matter since we aren't inside of Uno.Thems